tick/optim/model/base/model_hawkes.py

```diff
@@ -1,7 +1,10 @@
 # License: BSD 3 clause
 
 import numpy as np
+from scipy.sparse import sputils, csr_matrix
 
+from tick.optim.model.build.model import (ModelHawkesFixedSumExpKernLeastSqList,
```

Comment:
I find it cleaner than many `\`, I've seen it in another open source repo, probably scikit-learn